lib/ansible/modules/network/nxos/nxos_portchannel.py

```diff
@@ -203,6 +203,8 @@ def get_portchannel(module, netcfg=None):
         for pc in pc_table:
             if pc['group'] == module.params['group']:
                 portchannel_table = pc
+            elif module.params['group'].isdigit() and pc['group'] == int(module.params['group']):
```

Comment:
Did you test this change against all nxos platforms?  Just want to make sure since this area has been historically prone to idempotence issues that were fixed in `2.4`.

Reply:
@mikewiebe ... I've tested our matrix and it passed the tests. I will rerun and confirm.

Reply:
@rahushen Did you retest this?

Reply:
@mikewiebe @trishnaguha Yes, retested and it passed the tests.